openedx_hooks/signals/auth/v1/__init__.py

```diff
@@ -8,4 +8,4 @@
 REGISTER_USER = Signal(providing_args=["user", "registration"])
 
 # Signal that fires when a user logins in the platform
-LOGIN_USER = Signal(providing_args=["user", "site"])
+LOGIN_USER = Signal(providing_args=["user", "request"])
```

Comment:
Perhaps we only need the "user" parameter and get the request with crum.get_current_request() in the receiver if it is needed.